packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -284,16 +312,24 @@ export default connectStore(() => {
     createSelector(getVgpus, vgpus => map(vgpus, 'gpuGroup'))
   )
 
+  const getCanAdministrate = createSelector(
+    getCheckPermissions,
+    (_, props) => props.vm,
```

Comment:
Returns the `id` directly here, to avoid rechecking the permission if an unrelated property changes (e.g. `name_label`).